src/bandmap.c

```diff
@@ -826,6 +817,16 @@ void bandmap_show() {
     move(cury, curx);			/* reset cursor */
 
     refreshp();
+
+    gettimeofday(&tv, NULL);
```

Comment:
Maybe the bmdata_write_file should also be called from time_update(). Then we have concentrated all time dependent actions in one place. What do you think?